doc/Command-Reference.md

```diff
@@ -9551,9 +9642,19 @@ This command can be used to clear the counters for all queues of all ports. Note
 
 - Usage:
   ```
-  show queue counters [<interface_name>]
+  show queue counters [OPTIONS] [interface_name]
   ```
 
+- Parameters:
+  - _interface_name_: display counters for interface name only
+
+- Options:
+  - _-a,--all_: display all counters
+  - _-T,--trim_: display trimming counters only
```

Comment:
Does queuestat -c clears the trim counters as well?

Reply:
sonic-clear queuecounters as well

Reply:
@kperumalbfn please have a look at `test_clear_port_trim_counters` and `test_clear_queue_trim_counters`